docs/tools/vdb_table/data/apachesolr.json

```diff
@@ -113,8 +113,8 @@
     "comment": ""
   },
   "in_process": {
-    "support": "none",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://solr.apache.org/docs/9_4_0/core/org/apache/solr/client/solrj/embedded/EmbeddedSolrServer.html",
```

Comment:
I presume this means that `EmbeddedSolrServer` is something that can only be embedded in Java apps? If so, it makes sense to keep this marked as `"full"`, but update the `"comment"` field to state `"Java apps only"`